lk2nd/device/dts/msm8953/msm8953-xiaomi-vince.dts

```diff
@@ -22,23 +22,28 @@
 
 			qcom,mdss_dsi_td4310_fhdplus_video_e7 {
 				compatible = "xiaomi,td4310-fhdplus-e7";
-				touchscreen-compatible = "novatek,nt36525-i2c";
+				touchscreen-compatible = "syna,rmi4-i2c";
+				// touchscreen-compatible = "novatek,nt36525-i2c";
```

Comment:
I guess I should've noticed they're all the same in the last pr. If you always pick the syna, why novatek exists in the dts? Or they both exist but not tied to the panel?

Reply:
Yes they are not tied to the panel there are same panel with different touch variant so we choosen one touch variant as primary and the other can be enabled by the users.
https://wiki.postmarketos.org/wiki/Xiaomi_Redmi_5_Plus_(xiaomi-vince)#Touchscreen

In long term solution i2c probe, or dt overlays or an extra extlinux parameter could be better.

Reply:
I see, makes sense.